src/farmos.yaml

```diff
@@ -0,0 +1,43 @@
+---
+repoUrl: 'https://github.com/farmOS/farmOS'
+name: farmOS
+description: >-
+  farmOS is a web-based application for farm management, planning, and record
+  keeping. It is developed by a community of farmers, developers, researchers,
+  and organizations with the aim of providing a standard platform for
+  agricultural data collection and management.
+websiteUrl: 'https://farmOS.org'
+license: GPL-2.0
+sdgs: []
```

Comment:
Oh, there is no sdgs linked!

Reply:
Oh my bad ^^'

Reply:
We should probably also check for mandatory fields in the automatic PR-action 🤔